tappio/unicode.py

```diff
@@ -0,0 +1,5 @@
+import sys
+if sys.version_info.major == 3:
+    unicode = str
+else:
+    unicode = unicode
```

Comment:
Why not use `six.text_type`?

Reply:
Here I remind readers that Python 2 end-of-life is now only six months away. Here I am, contributing to a museum! ;-)